src/api/spec/routing/api_matcher_spec.rb

```diff
@@ -17,14 +17,14 @@
     it { expect(get("/public/configuration?format=#{format}")).to route_to(controller: 'public', action: 'configuration_show', format: format) }
 
     it {
-      expect(get("/public/build/project/repository/arch/package/file.#{format}")).
+      expect(get("/public/build/project/repository/arch/package/filename?format=#{format}")).
```

Comment:
Yes but I wanted to have it consistent: arch -> arch, package -> package and then filename -> file?
And it was anyway necessary to change the format ...